ipatests/test_integration/test_replica_promotion.py

```diff
@@ -47,6 +48,32 @@ def test_kra_install_master(self):
         assert(found > 0), result2.stdout_text
 
 
+def test_sssd_config_allows_ipaapi_access_to_ifp(host):
+    '''Tests that the sssd configuration allows the ipaapi user to access
+    ifp
+
+    :param host the machine on which to test that sssd allows ipaapi
+    access to ifp
+    '''
+    contents = host.get_file_contents(paths.SSSD_CONF, encoding='utf-8')
+    # Parse the sssd config as a config file
+    config = ConfigParser()
+    config.read_string(contents)
+
+    # Verify that there is an ifp section present
+    assert config.has_section('ifp')
+
+    # Verify that the ifp section contains an allow_uids item
+    ifp_items = config.items('ifp')
+    ifp_items_dict = {item[0]: item[1] for item in ifp_items}
+    assert 'allowed_uids' in ifp_items_dict
+
+    # Verify that the allow_uids item's value contains ipaapi
+    uids = [uid.strip() for uid in ifp_items_dict['allowed_uids'].split(',')]
+    assert 'ipaapi' in uids
+
```

Comment:
Only two blank lines are necessary.

Reply:
I believe this is addressed in commit adc5e44.